tests/test-form-data-boundary.js

```diff
@@ -105,8 +105,6 @@ tape('custom boundary with content-length', function (t) {
   request.post({
     url: server.url,
     headers: {
-      // @note `content-type` header is not re-calculated if `content-length`
-      // is present for formData body
       'content-type': 'multipart/anything; boundary=' + boundary,
```

Comment:
Add a similar test without `content-type`, it should generate content-type correctly.

Reply:
Added in b4003e0